content/information-distribution/signature.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Signature"
+weight: 4
+---
+
+Informo relies on cryptographic signatures in order to verify the integrity of
+data shared across the federation. These signatures are generated using
+asymmetric cryptography keys (`ed25519`, `hmac-sha256`, `hmac-sha512`, etc.),
+which private key solely owned by the end user.
+
+
+## Signed Matrix event
+
+### Event data
+
+Informo signed events are built using a common structure, inspired from [Matrix
+`m.room.encrypted`
+events](https://matrix.org/speculator/spec/HEAD/client_server/unstable.html#m-room-encrypted):
```

Comment:
Please use the right URL for Matrix's CS API specs, which is https://matrix.org/docs/spec/client_server/r0.4.0.html#m-room-encrypted in this case.